### ISSUE TITLE
Represent accuracy of intermediate geometry points

### ISSUE THREAD

**Issue by j-d-b** [usdot-jpo-ode/wzdx#232]
## Background

In WZDx v4.0 (current), the [WorkZoneRoadEvent](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/WorkZoneRoadEvent.md) has two properties for describing the accuracy of the start and end positions of the road event at a high level, `beginning_accuracy` and `ending_accuracy`. These properties can have a value of either `estimated` or `verified`, defined in the [SpatialVerification](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/SpatialVerification.md) enumerated type. Thus it is currently only possible to describe the accuracy of the first and last coordinates in the containing [RoadEventFeature](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/RoadEventFeature.md)'s `geometry` (array of coordinates).

## Possible new feature

It may be beneficial to add properties or restructure how spatial accuracy is represented to allow describing the accuracy of the intermediate coordinates between the start and end in a work zone road event with a LineString geometry. Consumers could then have more information to decide how to use the intermediate coordinates in snapping the road event to their map.

## Drawbacks

For the majority of current WZDx feed producing agencies, the only position that will be verified is the start. In some but fewer cases, the end will be as well. Adding the ability to represent the accuracy of the points in between adds complexity to the specification for the benefit of only a small number of users—maybe none immediately. Likely this feature should be addressed only once there are more producers who are able to provide verified the middle points in the road event.

**Comment by sergebeaudry** [comment]
Great suggestion.   Today some vendors have GPs enabled devices that could be use to report an intermediate coordinates. EX: a smart arrow can report a left arrow with its GPS coordinate using the 4.0 SwzDeviceFeed .  This could then be use to create a segment within the workzone where the road impact will be reported based on the Arrow board mode.  Same apply for DSL where speed is dropped at the work zone approach or alternate one-way road using AFAD or flagger GPs devices within a larger work zone.

I think we will need it down the road.  do we need it now or later?    Let see data consumer opinions on the subject.

**Comment by j-d-b** [comment]
Closing as low priority/lack of interest. The most important positions are the start and end of the event. Most consumers snap/map the line to their basement so the accuracy of the intermediate points is not critical. This can be reopened if needs change.
